### DISCUSSION TITLE
AR viewing quality is poor on iOS device compare to Android device

### DISCUSSION THREAD

**Discussion by NitinJ88** [google/model-viewer#4214] [Q&A]
### Description
AR viewing quality is poor on iOS device compare to Android device. Edges and color information is missing. 

#### Live Demo

https://glitch.com/edit/#!/model-viewer

Original model: 
![OrignalFile](https://user-images.githubusercontent.com/86343802/231139191-9ed68019-940b-43f3-aaf4-0092bf1a1a59.png)

iOS AR view:
![iOS_AR_View](https://user-images.githubusercontent.com/86343802/231139348-b52c224d-6044-49b1-a953-64a1bff39207.jpeg)

Android AR view:

### Version

- model-viewer: v2.1.1

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [x] WebXR
- [ ] SceneViewer
- [x] QuickLook

**Comment by NitinJ88** [comment]
Android View:
![Android_AR_View](https://user-images.githubusercontent.com/86343802/231141369-c89398fd-6889-49a1-8da1-828f0ff378c1.jpg)

**Reply by elalish** [reply]
This is our USDZ auto-generation not doing the best job. In cases like this, we recommend converting it to USDZ yourself and setting that as the `ios-src`. Unfortunately until Apple starts supporting glTF natively, it's difficult to get consistent translation and rendering (@grorg).
